components/select/theme.css

```diff
@@ -15,3 +15,17 @@
   border-radius: 0 4px 4px 0;
   min-height: 100%;
 }
+
+.validation-icon {
+  margin-top: -1px;
+}
+
+.has-error {
+  &:not(.is-inverse) .validation-text {
```

Comment:
I guess you can remove the not here if you specify what `is-inverse` color should be underneath. As long as it stays in this order, ofc.

Reply:
I could do that indeed, but then you end up with a color override instead of applying the right color at once...